backend/src/api/v1/dto/CategoryDTO.ts

```diff
@@ -0,0 +1,49 @@
+import BaseDTO from "./BaseDTO";
+import PlaylistDTO from "./PlaylistDTO";
+
+// https://github.com/bennyn/wlc-webapp/blob/wlc-webapp/0.0.36/src/main/java/com/welovecoding/tutorial/api/v1/dto/CategoryDTO.java
```

Comment:
aah ok, that explains the OO-pattern used here, I was wondering if you translated the Java Code.